src/wranglerAction.ts

```diff
@@ -424,6 +424,36 @@ async function wranglerCommands(
 					setOutput("pages-deployment-alias-url", pagesArtifactFields.alias);
 					setOutput("pages-deployment-id", pagesArtifactFields.deployment_id);
 					setOutput("pages-environment", pagesArtifactFields.environment);
+					// create github deployment, if GITHUB_TOKEN is provided
```

Comment:
Are github deploys only intended for pages? Is that not a thing for workers?

Reply:
For now, we'll just have it set up for pages for pages-action parity, but we can definitely support workers in the future